### PR TITLE
Vendor in latest buildah code

### PR DESCRIPTION
**Pull request by rhatdan**
Use the parsing code to properly setup podman build namespaces
Fixes support for network namespace and user namespace

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Need to wait for https://github.com/projectatomic/buildah/pull/761

**Comment by rhatdan** [comment]
Fixes https://github.com/projectatomic/libpod/issues/913

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @giuseppe @baude @mheon @wking PTAL

**Comment by mheon** [comment]
LGTM once the buildah PRs are in

File: cmd/podman/build.go [hunk 1]
```diff
@@ -145,22 +142,41 @@ func buildCmd(c *cli.Context) error {
 	}
 	defer runtime.Shutdown(false)
 
+	var stdout, stderr, reporter *os.File
```
Comment (TomSweeneyRedHat) [inline cmd/podman/build.go]: Does this address the blank line in the logs error?

Comment (rhatdan) [reply cmd/podman/build.go]: I think that is the goal.  Although I want @baude to confirm.

Comment (rhatdan) [reply cmd/podman/build.go]: I just wanted podman build to match up with what buildah bud is doing.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**